app/views/home/contact.html.erb

```diff
@@ -2,6 +2,6 @@
 
 <div>
   <% @contact_details.each do |k, v| %>
-    <div><%= "#{k}: #{v}"  %></div>
+    <div><%= "#{k.to_s.capitalize}: #{v}"  %></div>
```

Comment:
Maybe we should use more suggestive names instead of `k` and `v`

Reply:
I will have another PR for that.